easybuild/easyconfigs/n/netMHC/netMHC-4.0a.eb

```diff
@@ -25,6 +23,8 @@ checksums = [
     'd9938388c50bf49d748e5c1293435d7d9fae5d0a33303d795e767429508e368a',  # netMHC-4.0a.Linux.tar.gz
     'f3fb6d77ec11bc51de963e96fd75a0622f5098a2d1e37d750afa3709d5b876d1',  # netMHC-4.0a-data.tar.gz
 ]
+download_instructions = f"""{name} is proprietary software, but free for academics.
+It can be requested at https://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHC"""
```

Comment:
This easyconfig confuses me. 
What's going on with download_filename "data.tar.gz" but the filename is different? I suppose this should be mentioned clearly here.

Reply:
the link doesn't work for me anymore, looks like it should be `https://services.healthtech.dtu.dk/cgi-bin/sw_request?software=netMHC&version=4.0&packageversion=4.0a&platform=Linux` instead

Reply:
regarding `data.tar.gz`, that looks to be downloadable from: `https://services.healthtech.dtu.dk/services/NetMHC-4.0/data.tar.gz`
I guess the confusion was caused by https://github.com/easybuilders/easybuild-easyconfigs/pull/9739 removing the `source_urls`, which means `data.tar.gz` is no longer auto-downloaded

Reply:
I think f3fb169 fixes this. Once you have the netMHC/netMHCII source then the data file will download directly.

Reply:
yes, that works